frontend/assets/stylesheets/components/_action.scss

```diff
@@ -97,6 +97,29 @@
     cursor: default;
 }
 
+.action--snap {
```

Comment:
Not found anything much better than magic numbers here yet, at least not without affecting buttons as a whole so might leave this as is for now and then look into a refactor later. We need to rationalise our button styles anyway at some point.